config/swagger.yaml

```diff
@@ -373,6 +373,11 @@ definitions:
         type: array
         items:
           type: string
+    required:
```

Comment:
I see this starts https://www.pivotaltracker.com/story/show/153629617 
Is this PR meant to finish it as well?

Reply:
No, it doesn't - the Pivotal story has yet more work to be done. Here I only fix the crucial part of the API